### ISSUE TITLE
Problem when run: php artisan neo4j:migrate

### ISSUE THREAD

**Issue by divanski** [Vinelab/NeoEloquent#95]
> [ErrorException]  
>   Declaration of Vinelab\NeoEloquent\Eloquent\Builder::simplePaginate() should be compatible with Illuminate\Database\Eloquent\Builder::simplePaginate($perPage = NULL, $columns = Array, $pageName = 'page')

**Comment by Mulkave** [comment]
if you're using L5.1 make sure you have the latest version of 1.2.5 in composer by checking your `composer.lock` file and having `vinelab/neoeloquent` at 7bd1c6fb1acd3a66a1b68ea3e718bc473e6ecd38

**Comment by divanski** [comment]
I using L5.1 and latest v1.2.5 ... I change the line 316 on file Relation.php but problem still exist

**Comment by divanski** [comment]
I tried and https://github.com/Vinelab/NeoEloquent/commit/7e32b3b8d68f9ac8bfb0e05b260d1e178c6c45af , but still not work . The error is [ErrorException] array_diff(): Argument #2 is not an array ...

**Comment by lamoni** [comment]
Seeing this same issue, I'm on v1.2.5

```
lamoni@dev:/var/www# php artisan neo4j:migrate
 
  [ErrorException]                           
  array_diff(): Argument #2 is not an array  
                                             
```

Full stacktrace:

```
[2015-06-28 10:27:02] local.ERROR: exception 'ErrorException' with message 'array_diff(): Argument #2 is not an array' in /var/www/vendor/laravel/framework/src/Illuminate/Database/Migrations/Migrator.php:81
Stack trace:
#0 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(2, 'array_diff(): A...', '/var/www/vendor...', 81, Array)
#1 /var/www/vendor/laravel/framework/src/Illuminate/Database/Migrations/Migrator.php(81): array_diff(Array, Object(Illuminate\Database\Eloquent\Collection))
#2 /var/www/vendor/vinelab/neoeloquent/src/Vinelab/NeoEloquent/Console/Migrations/MigrateCommand.php(64): Illuminate\Database\Migrations\Migrator->run('/var/www/databa...', false)
#3 [internal function]: Vinelab\NeoEloquent\Console\Migrations\MigrateCommand->fire()
#4 /var/www/vendor/laravel/framework/src/Illuminate/Container/Container.php(502): call_user_func_array(Array, Array)
#5 /var/www/vendor/laravel/framework/src/Illuminate/Console/Command.php(149): Illuminate\Container\Container->call(Array)
#6 /var/www/vendor/symfony/console/Command/Command.php(259): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#7 /var/www/vendor/laravel/framework/src/Illuminate/Console/Command.php(135): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#8 /var/www/vendor/symfony/console/Application.php(878): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#9 /var/www/vendor/symfony/console/Application.php(195): Symfony\Component\Console\Application->doRunCommand(Object(Vinelab\NeoEloquent\Console\Migrations\MigrateCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#10 /var/www/vendor/symfony/console/Application.php(126): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#11 /var/www/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(94): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#12 /var/www/artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#13 {main}  

```

**Comment by Mulkave** [comment]
@divanski r u still facing issues after @lamoni's fixes?

**Comment by divanski** [comment]
I still have a problem

**Comment by Mulkave** [comment]
@divanski have you tried v1.2.6?

**Comment by lamoni** [comment]
@divanski Can you describe the problem or show outputs?  I've used my commit to run a couple different migrations on different projects I have in flight.  Are you sure you pulled a commit with the fix (it hasn't been merged yet as I still haven't had time to rebase with the latest 1.2.5)

**Comment by RomaGilyov** [comment]
Have the same issue Laravel version is 5.2.32 and NeoEloquent is 1.3.1

ErrorException: Declaration of Vinelab\NeoEloquent\Eloquent\Builder::simplePaginate() should be compatible with Illuminate\Database\Eloquent\Builder::simplePaginate($perPage = NULL, $columns = Array, $pageName = 'page', $page = NULL)

**Comment by bluerain75** [comment]
@RomaGilyov, should be resolved in NeoEloquent 1.3.2, that solved the same issue for me.
